### PR TITLE
Support Python 3

### PR DESCRIPTION
**Pull request by ofek**
### Additional Notes

Also added e2e

### CODE REVIEW

File: eventstore/datadog_checks/eventstore/eventstore.py [hunk 1]
```diff
@@ -36,16 +35,15 @@ def check(self, instance):
             raise CheckException('Invalid Status Code, {0} returned a status of {1}.'.format(url, r.status_code))
         # Unable to deserialize the returned data
         try:
-            parsed_api = json.loads(r.text)
+            parsed_api = r.json()
```
Comment (gmmeyer) [inline eventstore/datadog_checks/eventstore/eventstore.py]: does this raise the same error? I think so but it's good to confirm

File: eventstore/datadog_checks/eventstore/eventstore.py [hunk 2]
```diff
@@ -99,19 +97,25 @@ def check(self, instance):
...
+    def format_tag(cls, name):
+        """Converts the string to snake case from camel case"""
         # https://stackoverflow.com/questions/1175208/elegant-python-function-to-convert-camelcase-to-snake-case
         s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
         return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
-    def walk(self, json_obj, p=[], es_paths=[]):
+    def walk(self, json_obj, p=None, es_paths=None):
         """ Walk a JSON tree and return the json paths in a list """
+        if p is None:
```
Comment (gmmeyer) [inline eventstore/datadog_checks/eventstore/eventstore.py]: why did you change these?

File: eventstore/datadog_checks/eventstore/eventstore.py (hunk 1 above)
Comment (ofek) [reply eventstore/datadog_checks/eventstore/eventstore.py]: Yes

File: eventstore/datadog_checks/eventstore/eventstore.py (hunk 2 above)
Comment (ofek) [reply eventstore/datadog_checks/eventstore/eventstore.py]: We've made efforts to remove all mutable args

File: eventstore/datadog_checks/eventstore/eventstore.py (hunk 1 above)
Comment (ofek) [reply eventstore/datadog_checks/eventstore/eventstore.py]: actually no, py3 json has its own error `JSONDecodeError`

Comment (gmmeyer) [reply eventstore/datadog_checks/eventstore/eventstore.py]: ahh great!

File: eventstore/datadog_checks/eventstore/eventstore.py (hunk 2 above)
Comment (gmmeyer) [reply eventstore/datadog_checks/eventstore/eventstore.py]: sounds like a good change
